doc/man3/SSL_CTX_set_session_ticket_cb.pod

```diff
@@ -100,19 +100,54 @@ Same as B<SSL_TICKET_SUCCESS>, but a new ticket should be sent to the client.
 
 =back
 
+The return value can be any of these values:
+
+=over 4
+
+=item SSL_TICKET_RETURN_ERROR
+
+An error occurred.
+
+=item SSL_TICKET_RETURN_NO_TICKET
+
+Do not use a ticket (if one was available). Do not send a renewed ticket to the
+client.
+
+=item SSL_TICKET_RETURN_NO_TICKET_RENEW
+
+Do not use a ticket (if one was available). Send a renewed ticket to the client.
+
+If the callback does not wish to change the default ticket behaviour then it
+should return this value if B<status> is B<SSL_TICKET_EMPTY> or
+B<SSL_TICKET_NO_DECRYPT>.
+
+=item SSL_TICKET_RETURN_TICKET
```

Comment:
I would rename this to `SSL_TICKET_RETURN_USE`

Reply:
Done